solution/backend/regulations/templates/regulations/base.html

```diff
@@ -35,29 +35,42 @@
     </head>
     <body class="ds-base {% block body_class %}{% endblock %}">
         <div id="vue-app">
-            {% block pre_header %}{% endblock %}
+            {% block blocking_modal %}
+                <blocking-modal
+                    where-used="django"
+                >
+                    <i-frame-container
+                        src="{{ SURVEY_URL }}"
+                        title="Google Forms iframe"
```

Comment:
Is this just a lace holder text or does it not matter.

Reply:
it needed something as a title to pass a11y.  It's accurately descriptive since we're using the iFrame container to embed the survey.